awx/ui_next/src/components/AppContainer/AppContainer.jsx

```diff
@@ -132,6 +222,28 @@ function AppContainer({ i18n, navRouteConfig = [], children }) {
         {i18n._(t`Failed to retrieve configuration.`)}
         <ErrorDetail error={configError} />
       </AlertModal>
+      <AlertModal
+        title={i18n._(t`Your session is about to expire`)}
+        isOpen={timeoutWarning && sessionTimeout > 0}
+        showClose={false}
+        variant="warning"
+        actions={[
+          <Button
+            key="confirm"
+            variant="primary"
+            onClick={handleSessionContinue}
+          >
+            {i18n._(t`Continue`)}
+          </Button>,
+          <Button key="logout" variant="secondary" onClick={handleLogout}>
+            {i18n._(t`Logout`)}
+          </Button>,
+        ]}
+      >
+        {i18n._(t`You will be logged out in`)}{' '}
+        {Number(Math.floor(timeRemaining / 1000))}{' '}
+        {i18n._(t`seconds due to inactivity.`)}
```

Comment:
Could we wrap this entire message for translation instead of splitting it up into chunks?
```suggestion
        {i18n._(t`You will be logged out in ${Number(Math.floor(timeRemaining / 1000))} seconds due to inactivity.`)}

```

Reply:
Good idea. Fixed!